backend/globaleaks/handlers/accreditator/__init__.py

```diff
@@ -0,0 +1,158 @@
+from globaleaks.handlers.accreditator.services import accreditation, get_all_accreditation, persistent_drop, \
+    get_accreditation_by_id, update_accreditation_by_id, toggle_status_activate, activate_tenant, \
+    from_invited_to_request
+from globaleaks.handlers.base import BaseHandler
+from globaleaks.rest import requests, errors
+import logging
+import json
+
+class ConfirmRequestHandler(BaseHandler):
+    """
+    This manager is responsible for receiving accreditation requests and forwarding them to the accreditation manager
+    """
+    check_roles = 'any'
+    root_tenant_only = True
+    invalidate_cache = True
+
+    def post(self, accreditation_id: str):
+        body_req = self.request.content.read()
+        fiscal_code = self.request.headers.get(b'x-idp-userid')
+        request = self.validate_request(body_req,requests.SubmitAccreditation)
+        request['client_ip_address'] = self.request.client_ip
+        request['client_user_agent'] = self.request.client_ua
+        request['admin_fiscal_code'] = fiscal_code
+        try:
+            request['organization_institutional_site'] = json.loads(body_req).get('organization_institutional_site')
+        except Exception as e:
+            logging.debug(e)
+            request['organization_institutional_site'] = None
+        return from_invited_to_request(request, accreditation_id)
+
+class SubmitInstructorRequestHandler(BaseHandler):
+    """
+    This manager is responsible for receiving accreditation requests and forwarding them to the accreditation manager
+    """
+    check_roles = 'receiver'
+    root_tenant_only = True
+    invalidate_cache = True
+
+    def post(self):
+        body_req = self.request.content.read()
+        request = self.validate_request(
+            body_req,
+            requests.AccreditationInstructorRequest)
+        request['client_ip_address'] = self.request.client_ip
+        request['client_user_agent'] = self.request.client_ua
+        try:
+            request['organization_institutional_site'] = json.loads(body_req).get('organization_institutional_site')
+        except Exception as e:
+            logging.debug(e)
+            request['organization_institutional_site'] = None
+        return accreditation(request, is_instructor = True)
+
+class ToggleStatusActiveHandler(BaseHandler):
+    check_roles = 'any'
```

Comment:
SECURITY ISSUE: any authenticated user can access this handler regardless of their permissiona